src/common/dataHooks/useMediaObject.js

```diff
@@ -0,0 +1,45 @@
+import { useQuery } from '@tanstack/react-query'
+
+// FPCC
+import {
+  AUDIO,
+  IMAGE,
+  VIDEO,
+  AUDIO_PATH,
+  IMAGE_PATH,
+  VIDEO_PATH,
+} from 'common/constants'
+import api from 'services/api'
+
+export default function useMediaObject({ sitename, id, type }) {
```

Comment:
Any reason to pass a type here rather than having 3 separate data hooks? It looks like the type is always hard-coded

Reply:
Good call. Done :)